test/rdoc/test_rdoc_top_level.rb

```diff
@@ -162,6 +162,14 @@ def test_http_url
     assert_equal 'path_other/level_rb.html', other_level.http_url
   end
 
+  def test_path
+    assert_equal 'path/top_level_rb.html', @top_level.path
+
+    # Note: @options != @store.options
```

Comment:
Oh... Can we remove this comment by https://github.com/ruby/rdoc/pull/1332 ?

Reply:
I'll rebase and test once #1332 is merged 👍

Reply:
Merged!